lang/py/.gitignore

```diff
@@ -3,3 +3,6 @@
 build/
 lib/
 userlogs/
+avro/HandshakeRequest.avsc
+avro/HandshakeResponse.avsc
+avro/VERSION.txt
```

Comment:
In setup.py, these files are also copied into this package:

avro/interop.avsc
avro/tether/InputProtocol.avpr
avro/tether/OutputProtocol.avpr

Ignore them as well?